### PR TITLE
Comments

### PR DESCRIPTION
**Pull request by Fester2004**
add models and forms for  user_comments

### CODE REVIEW

File: Server/GGuide/models.py [hunk 1]
```diff
@@ -61,3 +61,15 @@ class FriendForm(forms.Form):
     email = forms.CharField(label='email', max_length=22)
 
 
+# class CommentsForm(forms.Form):
```
Comment (AndrewLvov) [inline Server/GGuide/models.py:64]: What is this ?

File: Server/GGuide/models.py [hunk 2]
```diff
@@ -61,3 +61,15 @@ class FriendForm(forms.Form):
...
+#     comment = models.CharField(label='text', max_length=250)
+
+
+class CommentsModel(models.Model):
```
Comment (AndrewLvov) [inline Server/GGuide/models.py:68]: I suggest not adding `Model` suffix to class name

File: Server/GGuide/models.py [hunk 3]
```diff
@@ -61,3 +61,15 @@ class FriendForm(forms.Form):
...
+    user_img = models.ImageField()
+    user = models.OneToOneField(User, on_delete=models.CASCADE)
+    article = models.ForeignKey(Article, on_delete=models.SET_NULL, null=True)
```
Comment (AndrewLvov) [inline Server/GGuide/models.py:71]: Can a comment be without an article ?

File: Server/GGuide/models.py (hunk 2 above)
Comment (Fester2004) [reply Server/GGuide/models.py:68]: ok

File: Server/GGuide/models.py (hunk 3 above)
Comment (Fester2004) [reply Server/GGuide/models.py:71]: No:) I will fix this mistake)

**Comment by AndrewLvov** [comment]
User uploaded files should not be committed to source control

**Comment by Fester2004** [comment]
> 
> 
> User uploaded files should not be committed to source control

?

**Comment by AndrewLvov** [comment]
What is `Server/profile_images/RobloxScreenShot20190922_135310505.png` ? How is it related to `Comments` model ?

**Pull request merged**